aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -1020,6 +1054,16 @@ Tensor& greater_equal_out(const Tensor& self, const Scalar& other, Tensor& resul
 Tensor greater_equal(const Tensor& self, const Scalar& other) { return self.ge(other); }
 Tensor& greater_equal_(Tensor& self, const Scalar& other) { return self.ge_(other); }
 
+TORCH_IMPL_FUNC(eq_Tensor_out)
+(const Tensor& self, const Tensor& other, const Tensor& result) {
+  eq_stub(device_type(), *this);
+}
+
+TORCH_IMPL_FUNC(eq_Scalar_out)
+(const Tensor& self, const Scalar& other, const Tensor& result) {
+  eq_stub(device_type(), *this);
+}
+
```

Comment:
Delete the lines below?

Reply:
Oops. I think I amended to the wrong commit.